webapps/console/components/SignInOrUp/SignIn.tsx

```diff
@@ -117,7 +117,6 @@ export const SocialLogin: React.FC<{ onSocialLogin: (type: string) => Promise<vo
               await onSocialLogin("github.com");
               redirectIfNeeded(router);
             } catch (error: any) {
-              console.log(JSON.stringify(error, null, 2));
               setError(handleFirebaseError(error));
```

Comment:
I think `handleFirebaseError()` already logs the error, so best to remove an apparently stray `console.log()`